### PR TITLE
Story4

### PR DESCRIPTION
**Pull request by AlicjaZub**

### CODE REVIEW

**Review by Alina1319 — APPROVED** [review]

File: script.js [hunk 1]
```diff
@@ -50,17 +51,32 @@ function playerTurn() {
...
     document.querySelectorAll('.box').forEach(element => {
-        element.removeEventListener('click',  boxClick)
+        element.removeEventListener('click', boxClick)
     })
 }
 
+function speedLvl() {
+    let minimumSpeed = 300;
+    if (speedTime === 300) {
+        speedTime = minimumSpeed;
```
Comment (mporam) [inline script.js]: why? this code currently says:
```
if speedTime is 300, make speedTime equal 300
```
?!

File: script.js [hunk 2]
```diff
@@ -50,17 +51,32 @@ function playerTurn() {
...
+
 function boxClick(e) {
     if (e.target.id !== colorPattern[0]) {
         toggleEndGameModal()
     } else {
         colorPattern.shift()
         if (colorPattern.length === 0) {
             timesRun = 1
+            level++
+            if (level !== 1 && level %2 !== 0) {
```
Comment (mporam) [inline script.js]: this feels like it should be its own function.

also missing a space before the `%`

File: script.js [hunk 3]
```diff
@@ -50,17 +51,32 @@ function playerTurn() {
...
+                counter++
+            }
+            speedLvl()
+            timesRun = timesRun - counter
```
Comment (mporam) [inline script.js]: ca use `-=` here

File: script.js [hunk 4]
```diff
@@ -50,17 +51,32 @@ function playerTurn() {
 
 function removeEventListener() {
     document.querySelectorAll('.box').forEach(element => {
-        element.removeEventListener('click',  boxClick)
+        element.removeEventListener('click', boxClick)
     })
 }
 
+function speedLvl() {
```
Comment (mporam) [inline script.js]: poorly named function, this doesnt describe what the function does

**Review by mporam — CHANGES_REQUESTED** [review]

File: script.js [hunk 5]
```diff
@@ -55,11 +55,17 @@ function removeEventListener() {
...
 }
 
-function speedLvl() {
-    let minimumSpeed = 300;
-    if (speedTime === 300) {
-        speedTime = minimumSpeed;
-    } else if (level % 4 === 0) {
+function levelIncrease() {
+    level++
+    if (level !== 1 && level %2 !== 0) {
```
Comment (mporam) [inline script.js]: still need a space before the `%`

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**